integreat_cms/cms/forms/linkcheck/edit_url_form.py

```diff
@@ -58,14 +53,12 @@ def clean(self, value: str) -> str:
             )
             self.validators.append(EmailValidator())
             self.error_messages["invalid"] = _("Enter a valid email address.")
-            value = f"mailto:{super().clean(email)}"
-        else:
-            if not value.startswith("tel:"):
-                logger.debug("Value %r is a normal link, enforcing URLValidator", value)
-                self.validators.append(URLValidator())
-                self.skip_url_fragment_validation = False
-            value = super().clean(value)
-        return value
+            return f"mailto:{super().clean(email)}"
+        if not value.startswith("tel:"):
+            logger.debug("Value %r is a normal link, enforcing URLValidator", value)
+            self.validators.append(URLValidator(schemes=["http", "https"]))
```

Comment:
Even if there were ftp links, I guess it would be ok to only allow them to be removed.
But I just checked the db, and there aren't any:

```
In [1]: from linkcheck.models import Url

In [2]: Url.objects.filter(url__startswith="ftp").exists()
Out[2]: False
```